### ISSUE TITLE
Properly handle issue-comments

### ISSUE THREAD

**Issue by lachmanfrantisek** [packit/packit-service#688]
In https://github.com/packit-service/packit-service/pull/687 I've used a manual, old way of matching of events to handlers for issue-comment.

 To make it properly, we need to:
- [x] save the propose-update run to the database and match the trigger object to it.
- [ ] use the matched trigger when retriggering via comments

Superseded by https://github.com/packit/packit-service/issues/1536

Benefit:
- Possibility to use issue comments for other jobs.
- Be able to have multiple triggers for `propose-downstream`.

**Comment by lachmanfrantisek** [comment]
Still a problem and relevant to #847

**Comment by lachmanfrantisek** [comment]
This will be much easier to implement once #905 is merged.

**Comment by lachmanfrantisek** [comment]
This would be tough work but would improve the code and will allow a lot of UX improvements. So, reopening, but not sure when we have some time for that.
